.github/workflows/ci.yml

```diff
@@ -0,0 +1,49 @@
+name: iroha-java CI
+
+on:
+  push:
+
+env:
+  BUILD_DIR: client/build
+  SONAR_TOKEN: ${{ secrets.SONAR_TOKEN }}
+  CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
+  CACHE_VERSION: v01
+  CACHE_PATHS: |
+    ~/.gradle/caches
+    ~/.gradle/wrapper
+jobs:
+  Linux:
+    strategy:
+      fail-fast: false
+    runs-on: ubuntu-16.04
+    steps:
+      - uses: actions/checkout@v2
+      - uses: actions/cache@v2
+        with:
+          path: ${{ env.CACHE_PATHS }}
+          key: ${{ runner.os }}-gradle-${{ hashFiles('**/*.gradle*', '**/gradle-wrapper.properties') }}
+          restore-keys: |
+            ${{ runner.os }}-gradle-
+      - name: Set up JDK 1.8
+        uses: actions/setup-java@v1
+        with:
+          java-version: 1.8
+      - name: Build with Gradle
+        run: |
+          ./gradlew --daemon
+          ./gradlew build --parallel --build-cache
+          ./gradlew jacocoTestReport
+      - name: Cleanup Gradle Cache
+        run: |
+          rm -f ~/.gradle/caches/modules-2/modules-2.lock
+          rm -f ~/.gradle/caches/modules-2/gc.properties
+      - name: Codecov
+        env:
+          CODECOV_TOKEN: ${{ secrets.CODECOV_TOKEN }}
```

Comment:
use codecov module https://github.com/marketplace/codecov, it doesn't need CODECOV_TOKEN
example: https://github.com/libp2p/cpp-libp2p/pull/112/files#diff-a2115d277b5ca5a2f09a999e53440839cf332b94da177f3d1766334555b0f7c6R40

Reply:
CI updated